docs/advanced_configuration.md

```diff
@@ -0,0 +1,173 @@
+# Advanced Configuration
+
+## Global vs Instance Based Configuration
+
+Which approach suits best your needs will depend on the architecture of your application and how do your users need to register and authenticate to it.
+
+If you have a multi-tenant application, or any application segmenation, where your users register and authenticate to each of these tenants or segments individuallly using different hostnames, or with different security needs, you need to go through [Instance Based Configuration](#instance-based-configuration).
+
+However, if your application is served for just one hostname, or else if your users authenticate to only one subdmain (e.g. your application serves www.example.com and admin.example.com but all you users authenticate through auth.example.com) you can still rely on one [Global Configuration](../README.md#configuration).
+
+If you are still not sure, or want to keep your options open, be aware that [Instance Based Configuration](#instance-based-configuration) is also a valid way of defining a single instance configuration and how you share such configuration across your application, it's up to you.
+
+
+## Instance Based Configuration
+
+All options that are valid for [Global Configuration](../README.md#configuration)
+are also valid for each instance of your application that needs to be configured in the following way:
```

Comment:
I think that users trying to use this feature without prior knowledge of how this is implemented would have some questions like: 'where should I put this code? Should it be in `config/initializers/webauthn.rb`? If that's right, should it be inside the block `Webauthn.configure` or outside?

To be honest, I'm not even sure if I know the answer to those questions :woozy_face: